download-vmlinux/run.sh

```diff
@@ -3,22 +3,15 @@
 set -uo pipefail
 trap 'exit 2' ERR
 
-source $(cd $(dirname $0) && pwd)/../helpers.sh
+. "$(cd "$(dirname "$0")" && pwd)/../helpers.sh"
 
-TEMP=$(getopt -o 'k:b:r:fos:ISid:lh' --long 'kernel:,build:,rootfs:,force,one-shot,setup-cmd,skip-image,skip-source:,interactive,dir:,list,help' -n "$0" -- "$@")
```

Comment:
As discussed offline, let's get a CI run scheduled to get this tested.

And, actually, it will likely need to happen in the [`libbpf/libbpf`](https://github.com/libbpf/libbpf) repository, I believe, because only it downloads a tarball (and uses this action).

For my understanding, though, is this removal based on your detection of dead code or based on being flagged by `shellcheck`?

Reply:
It was flagged by shellcheck